pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -458,6 +465,12 @@ func (s *Service) DeclarePluginRoles(ctx context.Context, ID, name string, regs
 			return err
 		}
 
+		for i := range r.Role.Permissions {
+			// Register plugin actions and their possible scopes for permission validation
+			s.permRegistry.RegisterPluginScope(r.Role.Permissions[i].Scope)
```

Comment:
I'm a bit confused why we need this one... given the only scope we have so far is `plugins:id:<id>` for `plugins.app:access`. But it is future proof.